src/ui/CommerceQuery/CommerceQuery.ts

```diff
@@ -54,6 +55,7 @@ export class CommerceQuery extends Component {
     this.options = ComponentOptions.initComponentOptions(element, CommerceQuery, options);
 
     this.bind.onRootElement(QueryEvents.doneBuildingQuery, this.handleDoneBuildingQuery);
+    this.bind.onRootElement(OmniboxEvents.buildingQuerySuggest, this.handleBuildingQuerySuggest);
```

Comment:
Could there be a situation the standalone search box is configured without query suggest?
If so, would the `buildQuerySuggest` event still be triggered in this case?

If not, is there another event we could use? Or should we add a new event for stand-alone search box interfaces that is guaranteed to be triggered?

Reply:
I just tested and the listing is added to the `plan` request sent by the standalone searchbox.